docs/integrations/sources/codefresh.md

```diff
@@ -0,0 +1,52 @@
+# Codefresh
+This connector integrates Codefresh with Airbyte, enabling seamless data synchronization for analytics and pipeline monitoring. It provides streams like agents, builds, audit, analytics etc.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `type` | `string` | Type.  |  |
+| `api_key` | `string` | API Key.  |  |
+| `account_id` | `string` | Account Id.  |  |
+| `account_name` | `string` | Account Name.  |  |
+| `report_name` | `string` | Report Name.  |  |
+| `report_granularity` | `string` | Report Granularity.  |  |
+| `report_date_range` | `array` | Report Date Range.  |  |
+| `entity_type` | `string` | Entity Type.  |  |
+| `entity_id` | `string` | Entity Id.  |  |
+| `start_date` | `string` | Start date.  |  |
+| `from` | `string` | From.  |  |
+| `to` | `string` | To.  |  |
+| `startdate` | `string` | startDate.  |  |
+| `enddate` | `string` | endDate.  |  |
```

Comment:
This is a lot of parameters can be simplify to start date and end date. Please refactor these parameters. If they need to have different format you must apply transformations/macros to sent to the API.
<img width="743" alt="image" src="https://github.com/user-attachments/assets/0d8683d2-4246-420f-9f46-1edfb5d99a53">

Reply:
account name, entity_id  and entity_type aren't used anywhere please remove these parameters.

Reply:
@marcosmarxm I deleted the unused parameters. but I cant simplify remove the parameters like startDate or From and To because builds enpoint expects the parameter to be called startDate and endDate and for audits enpoint its the From and To .

Reply:
You can have a single parameter called `start_date` and use to both streams.

Reply:
@marcosmarxm seems like I've made a mistake . in their docs the value of parameter `from` and `to` should be in miliseconds since 1970 but I dont know how it didnt gave me error as I was using diff datetime format. so should I change it to as they have said but then we cant have only one `start_date` for boths streams as they will have diff type. 
<img width="817" alt="Screenshot 2024-10-19 at 7 13 24 AM" src="https://github.com/user-attachments/assets/0ca7f239-4481-4d53-9b6b-41cc227eac8f">

Reply:
@marcosmarxm I've changed these paramters to accept miliseconds value. Just merge it now xd :)